### PR TITLE
Radios3

### PR DESCRIPTION
**Pull request by jasperswart**
eindelijk klaar. ik ben benieuwd wat je ervan vindt!

### CODE REVIEW

File: packages/asc-core/src/components/Radio/RadioInput.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  id: string
+  name: string
+  label: string
+  value?: string
+  defaultChecked?: boolean
+  disabled?: boolean
+}
+
+const Radio = styled.input.attrs<Props>(props => ({
+  type: 'radio',
```
Comment (janjaap) [inline packages/asc-core/src/components/Radio/RadioInput.ts]: Why do you need to define the styled component's attributes when the component already has props set?

File: packages/asc-core/src/components/Radio/RadioInput.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  id: props.id,
+  name: props.name,
+  value: props.value,
+  defaultChecked: props.defaultChecked,
+  disabled: props.disabled,
+}))`
+  cursor: pointer;
+  height: 28px;
```
Comment (janjaap) [inline packages/asc-core/src/components/Radio/RadioInput.ts]: Shouldn't this value come from a centralised theme?

File: packages/asc-core/src/components/Radio/RadioInput.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  left: 0;
+  margin: 0;
+  opacity: 0;
+  position: absolute;
+  top: 0;
+  width: 28px;
```
Comment (janjaap) [inline packages/asc-core/src/components/Radio/RadioInput.ts]: Shouldn't this value come from a centralised theme?

File: packages/asc-core/src/components/Radio/RadioLabel.ts [hunk 4]
```diff
@@ -1,24 +1,32 @@
...
-const Label = styled.label.attrs<Props>({
-  htmlFor: ({ htmlFor }: Props) => htmlFor,
-  disabled: ({ disabled }: Props) => disabled,
-})<Props>`
-  color: ${props => (props.disabled ? '#bebebe' : '#434343')};
+const Label = styled.label.attrs<Props>(props => ({
+  htmlFor: props.htmlFor,
+  disabled: props.disabled,
+}))<Props>`
+  color: ${({ theme, disabled }) =>
```
Comment (janjaap) [inline packages/asc-core/src/components/Radio/RadioLabel.ts]: Shouldn't the values, that are used for styling (like `16px`, `28px`, `4px`, `AvenirNextLTW01-Regular` and such) come from the theme configuration?

File: packages/asc-core/src/components/Radio/RadioLabel.ts [hunk 5]
```diff
@@ -1,24 +1,32 @@
...
   font-size: 16px;
   padding-left: 38px;
   line-height: 28px;
 
   &::before {
     border-radius: 50%;
-    border: ${props =>
-      props.disabled ? '1px solid #bebebe' : '1px solid #000'};
+    border: ${({ theme, disabled }) =>
+      `1px solid ${
```
Comment (janjaap) [inline packages/asc-core/src/components/Radio/RadioLabel.ts]: As far as I know styled components, nested template literals should be used in the `css` function of the `styled-components` package. 
Like so:
```jsx
border: ${({ theme, disabled }) => css(`1px solid ${
  disabled
    ? getThemeColor(theme, 'tint', 'level5')
    : getThemeColor(theme, 'tint', 'level7')
}`)};
```

File: packages/asc-core/src/components/Radio/RadioInput.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import styled from '../../styled-components'
+
+type Props = {
+  id: string
+  name: string
+  label: string
+  value?: string
```
Comment (janjaap) [inline packages/asc-core/src/components/Radio/RadioInput.ts]: Shouldn't a radio button always have a value?

File: packages/asc-core/src/components/Radio/RadioInput.ts [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import styled from '../../styled-components'
+
+type Props = {
```
Comment (janjaap) [inline packages/asc-core/src/components/Radio/RadioInput.ts]: Shouldn't this use the props that are defined in the `Radio` index file?

File: packages/asc-core/src/components/Radio/RadioLabel.ts [hunk 8]
```diff
@@ -1,24 +1,32 @@
...
-  color: ${props => (props.disabled ? '#bebebe' : '#434343')};
+const Label = styled.label.attrs<Props>(props => ({
+  htmlFor: props.htmlFor,
+  disabled: props.disabled,
+}))<Props>`
+  color: ${({ theme, disabled }) =>
+    disabled
+      ? getThemeColor(theme, 'tint', 'level4')
+      : getThemeColor(theme, 'tint', 'level5')};
   font-family: 'AvenirNextLTW01-Regular', verdana, sans-serif;
```
Comment (chrisvanmook) [inline packages/asc-core/src/components/Radio/RadioLabel.ts]: You can perhaps even use the Typography component...

File: packages/asc-core/src/components/Radio/RadioLabel.ts [hunk 9]
```diff
@@ -29,16 +37,19 @@ const Label = styled.label.attrs<Props>({
...
 
   &::after {
-    background-color: #0000;
-    border: 6px solid;
+    background-color: ${({ theme }) => getThemeColor(theme, 'tint', 'level7')};
+    border: 8px solid;
     border-radius: 50%;
-    color: ${props => (props.disabled ? '#bebebe' : '#000')};
+    color: ${({ theme, disabled }) =>
+      disabled
```
Comment (chrisvanmook) [inline packages/asc-core/src/components/Radio/RadioLabel.ts]: same: https://github.com/Amsterdam/amsterdam-styled-components/pull/8/files#r272020884

File: packages/asc-core/src/components/Radio/RadioLabel.ts (hunk 5 above)
Comment (janjaap) [reply packages/asc-core/src/components/Radio/RadioLabel.ts]: > Or perhaps even:
> 
> ```sass
> border: 1px solid ${({ disabled }) => getThemeColor(theme, 'tint',  disabled ? 'level5' : 'level7')}
> ```

Yes, better

File: packages/asc-core/src/components/Radio/RadioInput.ts (hunk 6 above)
Comment (timvanoostrom) [reply packages/asc-core/src/components/Radio/RadioInput.ts]: Maybe not if it's disabled?

**Pull request closed without merging**